roles/wordpress-setup/tasks/self-signed-certificate.yml

```diff
@@ -1,10 +1,19 @@
 ---
 - name: Generate self-signed certificates
-  shell: >
-    openssl req -subj "/CN={{ item.value.site_hosts[0].canonical }}" -new
-    -newkey rsa:2048 -days 3650 -nodes -x509 -sha256
-    -keyout {{ item.key }}.key -out {{ item.key }}.cert
+  shell: "openssl req -subj \"/CN={{ item.value.site_hosts[0].canonical }}\" -new \
+    -newkey rsa:2048 -days 3650 -nodes -x509 -sha256 \
+    -extensions req_ext -config <( \
```

Comment:
The `extensions` parameter is mandatory to have openssl x509 key generation take into account the subjectAltName parameter (it references the `req_ext` section l.10).